resend/templates/project_invitation.tmpl

```diff
@@ -0,0 +1,41 @@
+{{define "subject"}}Project invitation | ReleaseManager{{end}}
+
+{{define "textBody"}}
+Hey,\n\n
+You have been invited to join project {{.projectName}} in ReleaseManager.\n\n
+Use following link to join the project:\n
+{{.acceptLink}}\n\n
+Or reject the invitation by using the following link:\n
+{{.rejectLink}}\n\n
+If you haven't signed up for ReleaseManager, do so after you confirm the invitation.\n\n
+Happy releasing!\n
+ReleaseManager
+{{end}}
+
+{{define "htmlBody"}}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <table class="email-container" cellpadding="0" cellspacing="0" width="100%" style="font-family: Arial, sans-serif; max-width: 600px; margin: 0 auto; background-color: #ffffff; padding: 20px; border-radius: 8px; box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);">
+        <tr>
+            <td>
+                <h1 style="text-align: center; font-size: 24px; margin-bottom: 20px;">Project Invitation</h1>
+                <p style="font-size: 16px; line-height: 1.5;">Hey,</p>
```

Comment:
"Hey," feels weirdly informal and at the same time cold. 😄 🥶 

Consider removing the salutation altogether, as it's hard to get right.

No salutation is fine, professional and quite user-friendly, as it gets to the point, which in case of generated messages a good thing, I would say.